templates/comments_section.twig

```diff
@@ -1,4 +1,4 @@
-<div class="container" id="comments">
```

Comment:
I removed the `.container` since I prefer to its width be controlled by the [parent](https://github.com/greenpeace/planet4-master-theme/blob/9da984962ad54767037327c55f5a50909f7bcf37/templates/single.twig#L86).
Also I've checked and only the post template is the only template that uses tho comment_section.